package.json

```diff
@@ -2129,10 +2134,14 @@
         "requirejs": "^2.3.6",
         "rxjs": "^6.5.4",
         "rxjs-compat": "^6.5.4",
+        "safe-buffer": "^5.2.1",
         "sanitize-filename": "^1.6.3",
+        "save-dev": "^0.0.1-security",
```

Comment:
Just double checking whether this was required or whehter you accidentally ran `npm i save-dev` instead of `npm i --save-dev`

Reply:
oh that's probably a mistake. I'll remove it